Automattic/wp-calypso
# Adds an action required state to the DIFM Lite in progress screen

#### Changes proposed in this Pull Request

This will be merged into the parent PR: https://github.com/Automattic/wp-calypso/pull/59895

After checkout, we take users directly to the “Website Content” form. If they do not fill it at that time, we need to give them a path to return to the form. Once the user has paid for the product, we block access to Calypso pages by showing an “In-Progress” screen.

This page will have two states:
* Website Content not submitted: This should have a link to the Website Content form. (User action required) (Implemented in this PR)
* Website Content submitted: No user action is required here. (This was done via https://github.com/Automattic/wp-calypso/pull/60881)

#### Testing instructions



* Purchase DIFM Lite by proceeding through the flow at /start/do-it-for-me.
* After checkout, do not fill the Website Content form. Instead go to `/home/<site slug>`, where `<site slug>` is the site for which the purchase was made.
* Confirm that the screen looks like this:
![image](https://user-images.githubusercontent.com/5436027/153848750-79dd6121-f214-4048-b3d0-363897457ac8.png)
* Click on the "Add content to your website" button and fill out the form.
* If the form has been successfully submitted, you should see this screen:
![image](https://user-images.githubusercontent.com/5436027/153848924-97b58b86-7e06-4664-afb3-18833b3ef1db.png)
